stl_maps.ccp

```diff
@@ -6,8 +6,8 @@ class LearnMap{
 public :
 
 private:
-    void create_map();
-    void print_map();
+    void create_dict(void);
+    void print_map(void);
 };
 
```

Comment:
https://github.com/CrazyToExplore/STL/pull/3/commits/e3bf32e05463c591e701b1f58d7173adcf49043e

Has the changes